account_cutoff_base/models/account_cutoff_line.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2021 Akretion (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountCutoffLine(models.Model):
+    _name = "account.cutoff.line"
+    _inherit = "analytic.mixin"
+    _description = "Account Cut-off Line"
+
+    parent_id = fields.Many2one("account.cutoff", string="Cut-off", ondelete="cascade")
+    name = fields.Char("Description")
+    company_currency_id = fields.Many2one(
+        related="parent_id.company_currency_id",
+        string="Company Currency",
+        readonly=True,
+    )
+    partner_id = fields.Many2one("res.partner", string="Partner", readonly=True)
+    quantity = fields.Float(digits="Product Unit of Measure", readonly=True)
+    price_unit = fields.Float(
+        string="Unit Price w/o Tax",
+        digits="Product Price",
+        readonly=True,
+        help="Price per unit (discount included) without taxes in the default "
+        "unit of measure of the product in the currency of the 'Currency' field.",
+    )
+    price_origin = fields.Char(readonly=True)
```

Comment:
I fail to understand the purpose of this field, and why it is a `Char` but it's name suggests a price (thus it should be `Float` or `Monetary`)

Is this field even used? If so, what for? Should this be commented in the `help` attribute?

Reply:
It's a small note field that indicate where the price has been taken from. So it contains an invoice number, a PO number or an SO number.